.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files seem like they are generated after building / compiling the project. You can probably put these inside of a .gitignore file to reduce "clutter".